partner_email_check/__manifest__.py

```diff
@@ -3,7 +3,7 @@
 
 {
     "name": "Email Format Checker",
-    "version": "14.0.1.1.1",
+    "version": "15.0.1.1.1",
```

Comment:
@OpenCode As explained in migrating instructions, this kind of things must appears in migration commit, not in the pre-commit stuff one

Reply:
Moreover, when migrating, minor and patch identifiers should be reset. Thanks

Reply:
> @OpenCode As explained in migrating instructions, this kind of things must appears in migration commit, not in the pre-commit stuff one

I have a question about this point. If I don't change the version, pre-commit generate an error because it checks manifest version. Do I have to ignore it in the first step?

Reply:
> 

As said, explained in migrating link here above. See in details the 'Full process'. git commit --no-verify is the key.